### ISSUE TITLE
IME composition support

### ISSUE THREAD

**Issue by 0141KoppePan** [quasarframework/quasar#4482]
**Is your feature request related to a problem? Please describe.**
In vue.js v-model doesn't get updated during IME (Chinese, Japanese, Korean etc.) composition.But Quasar input v-model is not.

**Describe the solution you'd like**
Like Vue.js use compositionstart and compositionend events.
https://github.com/vuejs/vue/blob/dev/src/platforms/web/runtime/directives/model.js#L37-L38

**Describe alternatives you've considered**
``` diff
--- QInput.original.js	2019-06-27 09:32:39.437264878 +0900
+++ QInput.js	2019-06-27 09:39:49.679331055 +0900
@@ -21,6 +21,8 @@
 
     debounce: [String, Number],
 
+    compose: Boolean,
+
     maxlength: [Number, String],
     autogrow: Boolean, // makes a textarea
 
@@ -72,7 +74,10 @@
   },
 
   data () {
-    return { innerValue: this.__getInitialMaskedValue() }
+    return {
+      innerValue: this.__getInitialMaskedValue(),
+      composing: false
+    }
   },
 
   computed: {
@@ -115,6 +120,15 @@
       this.autogrow === true && this.__adjustHeight()
     },
 
+    __onCompositionStart () {
+      this.composing = true;
+    },
+
+    __onCompositionEnd () {
+      if (!this.composing) return;
+      this.composing = false
+    },
+
     __emitValue (val, stopWatcher) {
       const fn = () => {
         if (
@@ -126,7 +140,11 @@
 
         if (this.value !== val) {
           stopWatcher === true && (this.stopValueWatcher = true)
-          this.$emit('input', val)
+	  if (this.compose) {
+            !this.composing && this.$emit('input', val)
+          } else {
+            this.$emit('input', val)
+          }
         }
       }
 
@@ -157,7 +175,9 @@
         ...this.$listeners,
         input: this.__onInput,
         focus: stop,
-        blur: stop
+        blur: stop,
+        compositionstart: this.__onCompositionStart,
+        compositionend: this.__onCompositionEnd
       }
 
       if (this.hasMask === true) {
```

**Comment by rstoenescu** [comment]
Hi,

Thanks for reporting this!

Would you be so kind to contact me on Discord? Want some help on fully testing the fix (my commit above). Would be lovely if this happens before I release the final 1.0.0 this Sunday.

Thanks in advance!

**Comment by baryon** [comment]
From v1.15.6, The bug has reproduced. @rstoenescu 
v1.15.5 works

**Comment by dolphin836** [comment]
Still not available in the 2.12.1 version
